sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -139,6 +139,16 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    parent_parser.add_argument(
+        '--auth_user',
```

Comment:
Underscores are not appropriate in command line arguments - use --auth-user and --auth-password instead.